solutions/swb-reference/README.md

```diff
@@ -6,348 +6,728 @@ The 0.1.0 beta release is available for customers to test the newly re-architect
+      "value": "${iamPolicyDocument}"
+     },
+     {
+      "key": "InstanceType",
+      "value": "ml.t3.medium"      
+     },
+     {
+      "key": "AutoStopIdleTimeInMinutes",
+      "value": "0"
+     },
+     {
+       "key": "CIDR",
+       "value": "0.0.0.0/0"
+     }
+    ]
+}
+```
+In the response take note of the `id` that was returned. We'll refer to this value as `ENV_TYPE_CONFIG_ID`.
 
-5. For `terminatedEnvId` we need the id of an environment that has been terminated, Postman collection request `List Environments` does not show terminated environments ,so we need to save the id of a stopped environment before we terminate it.
+If you would like to launch a sagemaker notebook instance with a different instance type than `ml.t3.medium`, you can replace that value in the JSON above.
 
-    1. Create an environment by opening Postman Collection and select `Launch Environment` inside `environments` folder
-    
-    2. Click on the Body section of the request and fill the next values 
-        ```
-        {
-            "description": "test 123",
-            "name": "testEnv1",
-            "envTypeId": "<environment type id>",
-            "envTypeConfigId": "<environment type config id>",
-            "projectId": "<project id>",
-            "datasetIds": [],
-            "envType": "<environemnt type>"
-        }
-        ```
-        We can use the values from previous steps to fill the create request body
+### Associate Project to Environment Type Configuration
+In **SWBv2 Official** Postman Collection under **project** folder choose **Associate project with EnvTypeConfig** API.
 
-    3. Execute `Launch Environment` request and copy the id property from the response
-        ```
-        {
-            "id": "########-####-####-####-############",
-            "instanceId": "",
-            "cidr": "",
-            "description": "description",
-            "name": "environment Name",
-            "outputs": [],
-            ......
-        ```
+In the params tab set `projectId` parameter to the `PROJECT_ID` value from [Setup Project step](#setup-project).
+
+In the body tab set `envTypeId` parameter to the `ENV_TYPE_ID` value from [Retrieve Environment Type Id step](#retrieve-environment-type-id).
+
+In the body tab set `envTypeConfigId` parameter to the `ENV_TYPE_CONFIG_ID` value from [Create Environment Type Config step](#create-environment-type-config).
+
+Send **Associate project with EnvTypeConfig** request.
+
+PUT `{{API_URL}}/projects/:projectId/environmentTypes/:envTypeId/configurations/:envTypeConfigId/relationships`
+
+
+## Create new DataSet
+In **SWBv2 Official** Postman Collection under **datasets** folder choose **Create DataSet** API.
+
+During SWB deployment an S3 bucket for DataSets was created in your main account. Grab the name of that bucket from the CFN stack output (key named `DataSetsBucketName`) in the main account and construct the following API call
+
+POST `{{API_URL}}/datasets`
```

Comment:
I think this URL is out of date. Now, I think the datasets need to be created through an Project, meaning this must be done by a Project Admin. Meaning, we need a section for "create a user, associate user to project" section

Reply:
Create user section already exist, I will modify this section to point out we need a user with project permissions

Reply:
section updated